modules/javafx.graphics/src/main/java/javafx/scene/Scene.java

```diff
@@ -3630,6 +3630,7 @@ private void clearPDREventTargets() {
             currentEventTarget = currentEventTargets.size() > 0
                     ? currentEventTargets.get(0) : null;
             pdrEventTarget.clear();
+            pdrEventTargets.clear();  // clear to remove potentially removed nodes, see JDK-8264330
```

Comment:
We generally don't refer to the bug ID unless there is something particularly tricky about the fix (which isn't the case here).

Reply:
Removed this